python/ray/tests/test_queue.py

```diff
@@ -5,18 +5,19 @@
 from ray.experimental.queue import Queue, Empty, Full
 
 
-def test_queue(ray_start_regular):
-    @ray.remote
-    def get_async(queue, block, timeout, sleep):
-        time.sleep(sleep)
-        return queue.get(block, timeout)
+@ray.remote
+def get_async(queue, block, timeout, sleep):
+    time.sleep(sleep)
+    return queue.get(block, timeout)
 
-    @ray.remote
-    def put_async(queue, item, block, timeout, sleep):
-        time.sleep(sleep)
-        queue.put(item, block, timeout)
 
-    # Test simple usage.
+@ray.remote
+def put_async(queue, item, block, timeout, sleep):
+    time.sleep(sleep)
```

Comment:
While you're at it, do you think you could remove this reliance on `sleep`? This is at best going to inflate the testing time and at worst a recipe for flaky tests. We deal with this elsewhere in the tests by using a `SignalActor` - maybe you can copy from there.